collateral-info.yaml

```diff
@@ -713,9 +759,49 @@ ilks:
     rwa-stability-fee-payment-frequency: Monthly
     rwa-geographical-exposure: US
     rwa-vault-closure: Upon governance request
+  RWA010-A:
+    <<: *BASE
+    <<: *RWA010
+    <<: *STATUS_ACTIVE
+    id: RWA010-A
+    shortname: BBT1DRP
```

Comment:
Should we stick with the existing conventions for DROP values and name them something like: BBT1-DROP?

Reply:
I'd be tempted to do this even if the collateral symbol is technically different, just because it'll be more consistent for the non-technical audience.

Reply:
Keeping the BBT1-DROP reco.